src/api/AutonomousCars.Api/AutonomousCars.Api.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <InvariantGlobalization>true</InvariantGlobalization>
+    <UserSecretsId>a85b90dd-47aa-4542-9f93-acfbdbff43a9</UserSecretsId>
```

Comment:
à quoi il correspond cet Id?

Reply:
C'est un id auto-généré, il sert à identifier les secrets du projets dans le registre global il me semble (cf [Microsoft Learn - Safe storage of app secrets in development in ASP.NET Core - Enable secret storage](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-8.0&tabs=windows#enable-secret-storage))